shell/platform/tizen/tizen_renderer_evas_gl.h

```diff
@@ -41,6 +41,7 @@ class TizenRendererEvasGL : public TizenRenderer {
                           int32_t angle) override;
   void SetRotate(int angle) override;
   void SetPreferredOrientations(const std::vector<int>& rotations) override;
+  bool IsSupportedExtention(const char* name) override;
```

Comment:
IMO, This function is going to be called by string literal, so It will be ok now.

Reply:
Well... but `egl_extention_str_` is a std::string too. I think it would be nice to match it with a `egl_extention_str_`

Reply:
Yes, as you said, ``egl_extention_str_`` is a string, but it's only internal cache of extensions, so I wonder if there is a need to match it. From my point of view, IsSupportedExtention function seems to be called by literal, so there is no need to manage memory when using it, so it would not be bad to leave it for very tiny efficiency.